frontend/src/components/Button/Button.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import '../../../../styles/Button.css';
+
+function Button(props) {
```

Comment:
```suggestion
function Button( { buttonClassName, buttonText } ) {
```
Предлагаю сразу делать деструктуризацию пропсов